pkg/certificate/manager.go

```diff
@@ -117,9 +118,16 @@ func (m *Manager) start(ctx context.Context, mrcClient MRCClient) error {
 func (m *Manager) handleMRCEvent(mrcClient MRCClient, event MRCEvent) error {
 	switch event.Type {
 	case MRCEventAdded:
-		mrc := event.MRC
+		mrc := event.NewMRC
+
+		// ignore add event if MRC has no state
```

Comment:
What are the use cases for the MRC having no state?

Reply:
MRCs with no state are not in use. This allows the user to create an MRC without it automatically triggering a rotation.